plugin/package.json

```diff
@@ -28,14 +28,13 @@
     "pre-commit": "yarn run pretty-quick --staged --no-restage --bail --pattern ['**/*.{ts,tsx,scss,json}', '!src/kiali'] && npm run lint:precommit"
   },
   "dependencies": {
-    "@openshift-console/dynamic-plugin-sdk": "^1.3.0",
-    "@patternfly/patternfly": "5.1.0",
-    "@patternfly/react-charts": "7.1.1",
-    "@patternfly/react-core": "5.1.1",
-    "@patternfly/react-icons": "5.1.1",
-    "@patternfly/react-styles": "5.1.1",
-    "@patternfly/react-table": "5.1.1",
-    "@patternfly/react-topology": "5.2.1",
+    "@openshift-console/dynamic-plugin-sdk": "1.0.0",
```

Comment:
According to https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/README.md#openshift-console-versions-vs-sdk-versions, the version fully compatible with Openshift Console 4.15 is the version `1.0.0`. For this reason I prefer to downgrade to avoid unexpected behaviours.